blkin-lib/zipkin_c.c

```diff
@@ -92,6 +92,20 @@ int blkin_init_new_trace(struct blkin_trace *new_trace, const char *service,
 	return res;
 }
 
+int blkin_init_root_info(struct blkin_trace_info *trace_info)
+{
+	int res;
+	if (!trace_info) {
+		res = -EINVAL;
+		goto OUT;
+	}
+	trace_info->span_id = trace_info->trace_id = random_big();
+	trace_info->parent_span_id = 0;
+
+OUT:
+	return res;
+}
+
```

Comment:
> This may or may not be the same as the trace id.

interesting, thanks. i would definitely prefer we make less calls to rand, so i'm okay with having `blkin_init_trace_info()` and `blkin_init_new_trace()` always initialize span_id=trace_id